basic/minimal-e-commerce/init.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+scrypto build && \
+resim reset
+
+export owner_account=$(resim new-account)
+export owner=$(echo $owner_account | grep "Account component address:" | cut -d: -f2 | xargs)
```

Comment:
I'm on MacOS and running this command, the owner environment variable gets the value: `020d3869346218a5e8deaaf2001216dc00fcacb79fb43e30ded79a Public key`. 

Notice the "`Public key`" bit at the end? This has stopped this script from running on my machine. Any chance you could look into that? 

This might potentially be an issue with other environment variables, but I have not checked.

Reply:
This should now be fixed. I used regex for more consistency.